openpgp.d.ts

```diff
@@ -643,8 +630,9 @@ export interface FullKeyOptions {
   subkeys: KeyOptions[]; // required unlike KeyOptions.subkeys
 }
 
-export interface Keyid {
+export class KeyId {
```

Comment:
This implies that there is an `openpgp.KeyId` class that you can instantiate, which is not the case, so this should remain an interface. Also, the name of the (unexported) class is `Keyid`; it we wanna change this I think we should change the name of the class too, to `KeyID`. But for now I would keep it as is; the name of an interface doesn't really matter that much.